sycl/include/CL/sycl/accessor.hpp

```diff
@@ -409,6 +424,8 @@ class image_accessor
   // Available only when: (accessTarget == access::target::image || accessTarget
   // == access::target::host_image) && accessMode == access::mode::write ||
   // accessMode == access::mode::discard_write
```

Comment:
Please, update the condition in the comment.

Reply:
added the requested changes.